### ISSUE TITLE
Add documentation about PSReadLine not loaded when screen reader is detected

### ISSUE THREAD

**Issue by daxian-dbw** [MicrosoftDocs/PowerShell-Docs#4659]
Issue Details
-----------------

PR https://github.com/PowerShell/PowerShell/pull/10385 makes PowerShell Core to skip auto-loading `PSReadLine` on Windows if the [NVDA screen reader](https://www.nvaccess.org/download/) is active.
This is because `PSReadLine` doesn't work well with the screen reader as of today. The default rendering and formatting of PowerShell Core on Windows works with NVDA properly.

This change will be ported to Windows PowerShell, so 5.1 document also needs to be updated.

Version(s) of document impacted
------------------------------
- [x] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [x] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

**Comment by phiter** [comment]
Is it possible to disable this behavior? I use NVDA only to test my website, not for general usage on the pc, so I don't want to see this warning every time I start PowerShell.

**Comment by tyeth** [comment]
Please make this clearer to the end user, it's almost unsearchable (nothing listed in PsReadline issues), and check if the screen reader is active, not just installed or running. I similarly have installed screen readers for accessibility testing, but only run them when needed (once in a blue moon). My PSReadLine has been disabled for years now and it's very unnecessary as I never have a screen reader running.

**Comment by tyeth** [comment]
> Please make this clearer to the end user, it's almost unsearchable (nothing listed in PsReadline issues), and check if the screen reader is active, not just installed or running. I similarly have installed screen readers for accessibility testing, but only run them when needed (once in a blue moon). My PSReadLine has been disabled for years now and it's very unnecessary as I never have a screen reader running.
> 
> Worse still I had the **portable** versions of NVDA (2018.4.1 i think, but had two copies, one with custom plugin I was developing), both of which had no permanent installation. After deleting both copies and removing matching files from the windows prefetch folder I no longer get the warning about using a screen reader and PsReadLine is automatically imported.
> 
> @SteveL-MSFT you seem to be the puppet-master here and on PsReadLine, could you poke the right people please 🙏
> #4659 #5428

My bad, just retested with latest portable NVDA (2019.3.1) and when open PsReadline is not loaded, but once NVDA portable is closed, loading powershell loads PsReadline as expected.
